spec/components/review_draft/apply_draft/view_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ReviewDraft::ApplyDraft::View do
+  alias_method :component, :page
+
+  describe "sections that appear for application draft trainee" do
+    before do
+      render_inline(described_class.new(trainee: trainee))
+    end
+
+    context "when the trainee is on the provider-led route", 'feature_routes.provider_led_postgrad': true do
+      let(:trainee) { create(:trainee, :provider_led_postgrad, :with_placement_assignment, :with_apply_application) }
+
+      it "renders the correct sections" do
+        expect(component).to have_text("Course details")
+        expect(component).to have_text("Trainee data")
+        expect(component).to have_text("Placement details")
+        expect(component).to have_text("Training details")
+      end
+    end
+
+    context "when the trainee is on the school direct route", 'feature_routes.school_direct_salaried': true do
```

Comment:
We should also make sure it renders correctly for `school_direct_tuition_fee` trainees - see code snippet for other spec.

Reply:
cheers